### ISSUE TITLE
Sprint 1 – Scrum Meeting (2-12-2016)

### ISSUE THREAD

**Issue by mewind** [mewind/RSDSupermarketPOS#1]
Karcheng, may I know how is your progression on the login page?

**Comment by lkc442012** [comment]
Yesterday i have linked the database with the user login page. Hence, user are able to login to the system if their username and password are correct. Besides, i have also finished the "remember me" function that allow the system to save user's username and password so they don't need to retype it again on the next login,

**Comment by mewind** [comment]
So may I know whether the login function had fully working with the users stored in database?

**Comment by lkc442012** [comment]
Yes. I have try to create an user named "Gabriel" with password "54321", and i can log in successful since it is already existed in the database.

**Comment by mewind** [comment]
Good progression! So at this moment we had done the basic login page that link with the database. So for the "remember me " function , is that work properly that it will remember the username and password even the user had logged out ?

**Comment by lkc442012** [comment]
Actually there have some small problem in this part of code. "Remember me" function do not work if the user quits the program and rerun it again...

**Comment by huylin25** [comment]
i thought we do not expect the users to close the whole system because the system should be work all the day, so i think it should be ok.

**Comment by mewind** [comment]
Ya HuyLin is right. We should not allow the normal user to close the whole program. So it should be normal if the remember me didn't work after the whole program was closed. So KarCheng do you have any problem?

**Comment by lkc442012** [comment]
Other parts should be okay. So now Raymond and I will focus on the user's fail attempt login.

**Comment by juzplayplay** [comment]
I wish to implement an additional feature for login function. If an user failed to login after 3 attempts, an error message will prompt out and indicate that the user should wait for 3o seconds to try next attempt. How's that sounds?

**Comment by mewind** [comment]
Sounds a great idea to protect user's account ! So at this moment do you guys have any problem that need to be further discussed on the development of this feature ?

**Comment by juzplayplay** [comment]
This is an idea that I just had a moment ago. Kar Cheng and I will try implement it today and we will report the progress again tomorrow.

**Comment by mewind** [comment]
Alright ! Thanks Karcheng and Raymond, I am looking forward for this nice feature that can fulfill user story by protecting the account !

**Comment by mewind** [comment]
So HuyLin how's your progression? Do you have any problem?

**Comment by huylin25** [comment]
Yesterday i have done the view member details and edit details part. I also done the validation part as I try to edit the existing member details, i can successfully to edit the details of a member to database

**Comment by mewind** [comment]
Wow! We've done half of the membership module within a few days , thanks HuyLin for your effort. So until now do you have any problem on the View Member Details function and Edit Member Details function?

**Comment by huylin25** [comment]
There is no problem of that. I'm going to do the search member function in the membership module by today.

**Comment by mewind** [comment]
What kind of attribute you'll allow the user to search for a particular member ?

**Comment by huylin25** [comment]
i think i will allow them to search a member by using username and user ID, and then it will return the records that match with the value in the textbox.

**Comment by mewind** [comment]
Seems everything planning quite well. So I will leave them to you guys , we will further discuss about them tomorrow. In conclusion, KarCheng and Raymond will be working on Fail Login Attempt function and HuyLin will start developing the Search member function in Membership Management module. 

See you guys tomorrow! Thanks !

**Comment by huylin25** [comment]
Currently I have no problem on these 2 function. Today I will start to work with Member Registration function to let the login user to register themselves as the supermarket user.

**Comment by mewind** [comment]
Can you briefly explain on what will focus on later when developing this function?

**Comment by huylin25** [comment]
I will also include the validation for each field of the registration form to make sure the new member can be registered successfully. There will also some error messages when the user left the field empty or enter wrong value.

**Comment by mewind** [comment]
Do you remember to include the reset function to let user to reset all the values for the ease of error recovery ?

**Comment by huylin25** [comment]
Oh ya, It is also one of the requirement suggested in one of the user story, alright I will also be adding this into the function to reset all the fields.

Thanks for reminding!

**Comment by mewind** [comment]
You're welcome ! I think that's coming to the end of the meeting today. So letbme briefly conclude the outcome today, Karcheng and Raymond will be working on the Fail Login Attempt function and HuyLin will try to develop the User Registration of the Membership Management Module. 

All the best guys ! See you tomorrow!
